src/nav/nav.jsx

```diff
@@ -10,17 +14,67 @@ const linksWordsList = linksWord.map(function (link) {
         </Link>
     );
 });
-const linksPhoto = ['person.svg', 'magnifier.svg', 'heart.svg', 'shoppingCart.svg']
-const linksPhotoList = linksPhoto.map(function (link) {
-    return (
-        <Link to=''>
-            <img
-                src={`/photos/${link}`}
-                alt={link} />
-        </Link>
-    );
-});
+const navLinks = [
+    {
+        path: 'person',
+        extraClass: false
+    },
+    {
+        path: 'magnifier',
+        extraClass: false
+    },
+    {
+        path: 'heart',
+        extraClass: false
+    },
+    {
+        path: 'shoppingCart',
+        extraClass: true
+    },
+]
+
+
+
+const LinksPhotoList = ({ links, handleOpenModal }) => {
+    return links.map(link => {
+        return (
+            <button onClick={link.extraClass ? handleOpenModal : () => { }} className={link.extraClass ? 'navig_item shopping_cart' : 'navig_item'}>
+                <img
+                    src={`/photos/${link.path}.svg`}
+                    alt={link.path} />
+            </button>
+        );
+    })
+
+};
 const Nav = () => {
+    const [totalCost, setTotalCost] = useState(0)
+    const [showModal, setShowModal] = useState(false)
+    const products = useSelector((state) => state.product.products)
+    const dispatch = useDispatch()
+
+    const deleteProd = (id) => {
+        dispatch(deleteProduct(id))
+    }
+    useEffect(() => {
+        let totalProductsCost = 0;
+        products.forEach((product) => {
+            const costString = product.info.cost;
+            const costWithoutDots = costString.replace(/\./g, '');
+            const cost = parseInt(costWithoutDots) * parseInt(product.number);
+
+            totalProductsCost += cost;
+        })
```

Comment:
1. use number type for your `product.cost` as well as for `product.number`
2. it's also better to call it `product.quantity` instead of `product.number`
3. there is a simpler way to calculate this
```ts
const totalCost = products.reduce((totalCost, product) => totalCost + product.cost * product.number, 0)
```